src/App.js

```diff
@@ -286,9 +294,12 @@ const returnDefaultComponent = (Component, props) => {
   const reduxState = store.getState();
 
   if (reduxState.isVersionAllowed === undefined) {
+
+    helpers.loadStorageState();
+
     // We already handle all js errors in general and open an error modal to the user
     // so there is no need to catch the promise error below
-    version.checkApiVersion(reduxState.wallet);
```

Comment:
How will this work on the wallet service?

Reply:
After some though I reversed to how it was originally, the `loadStorageState` is enough to load the server url into the lib config.
And the "check version" will run right before going into loading_addresses.